Week_6/homework/backend/routes/todo.js

```diff
@@ -0,0 +1,55 @@
+/** Module for handling users */
+const express = require("express");
+const cors = require("cors");
+
+// Define route and middlewares
+const todo = express.Router();
+todo.use(cors());
+todo.use(express.json());
+
+  todo.post("/", async (req, res) => {
+    const todos = db.collection("todos");
+    const body = req.body;
+
+    const doc = await db.collection('todos').add({
+      todo: body.todo,
+      email: body.email,
+    });
+
+    const doc2 = await db.collection('todos').doc(doc.id).update({
+      todo: body.todo,
+      email: body.email,
+      uid: doc.id
+    });
+
+    res.status(200).send("OK")
```

Comment:
You are currently returning text; shouldn't you return the JSON from doc2? As a result, you are getting a 

`SyntaxError: Unexpected token 'O', "OK" is not valid JSON` on the frontend side when you try to convert this text to a json to console log.